Extensions/ServiceNow/Src/vss-extension.json

```diff
@@ -4,8 +4,8 @@
   "name": "ServiceNow change request management",
   "publisher": "ms-devlabs",
```

Comment:
should not the publisher be microsoft?

Reply:
Should be ms-vscs-rm ... the publisher registered on Marketplace with a display name as Microsoft